src/types.rs

```diff
@@ -554,15 +575,34 @@ impl From<Term> for Context {
 /// [`Operator`]: struct.Operator.html
 #[derive(Debug, PartialEq, Eq, Hash, Clone)]
 pub enum Term {
-    /// A concrete but unspecified `Term` (e.g. `x`, `y`)
+    /// A concrete but unspecified `Term` (e.g. `x`, `y`).
+    /// See [`Variable`] for more information.
+    ///
+    /// [`Variable`]: struct.Variable.html
     Variable(Variable),
-    /// An [`Operator`] applied to zero or more `Term`s (e.g. (`f(x, y)`, `g()`)
+    /// An [`Operator`] applied to zero or more `Term`s (e.g. (`f(x, y)`, `g()`).
     ///
     /// [`Operator`]: struct.Operator.html
     Application { op: Operator, args: Vec<Term> },
 }
 impl Term {
-    /// Describe the Term as a human-readable string
+    /// Describes the `Term` as a human-readable string.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use term_rewriting::{Signature, Term};
+    /// let mut sig = Signature::default();
+    /// let var = sig.new_var(Some("Z".to_string()));
+    /// let op = sig.new_op(0, Some("A".to_string()));
+    ///
+    /// let var_term = Term::Variable(var);
+    /// let args = vec![];
+    /// let op_term = Term::Application{op, args};
+    /// assert_eq!(var.display(&sig), "Z");
+    /// assert_eq!(var.display(&sig), var_term.display(&sig));
+    /// assert_eq!(op_term.display(&sig), "A()");
+    /// ```
```

Comment:
I don't think this test passes anymore given recent updates to how we display a `Variable`.

Also, it might be good to construct a somewhat more complex term and show how that displays (e.g. `A(B z_)`). rather than run multiple tests.